internal/cmd/schema-registry/command_cluster_describe.go

```
@@ -104,22 +116,23 @@ func (c *command) clusterDescribe(cmd *cobra.Command, _ []string) error {
 		mode = "<Requires API Key>"
 	}
 
-	query := schemaCountQueryFor(cluster.Id)
+	query := schemaCountQueryFor(clusters[0].GetId())
 	metricsResponse, httpResp, err := c.V2Client.MetricsDatasetQuery("cloud", query)
 	unmarshalErr := ccloudv2.UnmarshalFlatQueryResponseIfDataSchemaMatchError(err, metricsResponse, httpResp)
 	if unmarshalErr != nil {
 		return unmarshalErr
 	}
 
-	freeSchemasLimit := int(cluster.MaxSchemas)
-	if cluster.Package == essentialsPackageInternal {
+	freeSchemasLimit := defaultSchemaLimitEssentials
+	if strings.ToLower(clusterSpec.GetPackage()) == essentialsPackage {
 		user, err := c.Client.Auth.User(context.Background())
 		if err != nil {
 			return err
 		}
 		prices, err := c.Client.Billing.GetPriceTable(context.Background(), user.GetOrganization(), streamGovernancePriceTableProductName)
 		if err == nil {
-			priceKey := getMaxSchemaLimitPriceKey(cluster.Package, cluster.ServiceProvider, cluster.ServiceProviderRegion)
+			internalPackageName, _ := getPackageInternalName(clusterSpec.GetPackage())
```

Why are we ignoring the error?

---

cc @Anshul-Goyal